### ISSUE TITLE
Encryption related documentation seems to be irrelevant 

### ISSUE THREAD

**Issue by lzzy12** [simolus3/moor#701]
I am trying to keep my app's database encrypted, however I found the documentation to be quite irrelevant for me. It tells to remove the moor_flutter dependency, but I never added any such dependency and it suggests me to replace the use of FlutterExecuter with EncryptedExecuter but I never used that class in my code base. I would like some guidance in what am I missing?

**Comment by simolus3** [comment]
Yeah, those docs are a bit outdated. When using the encrypted version of moor, you can drop the `moor_ffi` dependency and replace `VmDatabase` with `EncryptedExecutor`.

In the future (hopefully soon) we will have an encrypted version of `moor_ffi` that doesn't require so much work.
